### PR TITLE
Add logo

### PR DESCRIPTION
**Pull request by robinandeer**
I'm sorry but we can't have an official Clinical fork of a repo without a logo :stuck_out_tongue_closed_eyes:

I do believe it *actually* say something about it in our AMSystems. I would dig it out but, you know, it's AMSystems, so I ask you take my word for it instead :wink:

### CODE REVIEW

**Comment by vezzi** [comment]
@robinandeer I really like the alternative one!!

**Comment by guillermo-carrasco** [comment]
I swear to God I was going to propose you to design a log today!!! I just imagined you were doing that already when you started asking about standard colors for bases :-P

I like very much the alternative one as well :-) <3

Thank you thank youuu!

**Comment by robinandeer** [comment]
Yay!!!
.
.
.
.
it passes travis! :smirk:

No, but thank you for the vote of confidence! :grinning: I'm trying your reward idea (designing a logo) because I have a lot of boring tasks to get through today...

File: requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+-r requirements.txt
```
Comment (guillermo-carrasco) [inline requirements-dev.txt]: will this file be used when you execute `python setup.py develop`? Or you have to make it explicit somehow?

Comment (robinandeer) [inline requirements-dev.txt]: No. I don't know of any such convention. The closest I've read about is ``test_requires`` argument for when you run ``python setup.py test``, right?

I usually separate dependencies into 1. general pip, 2. conda (FTW), 3. development. But then just install them for development using ``conda install --file requirements/conda.txt && pip install -r requirements/development.txt``.

Reference: https://github.com/robinandeer/chanjo/tree/master/requirements

Comment (guillermo-carrasco) [inline requirements-dev.txt]: This is really nice! 

Thanks!

File: README.md [hunk 2]
```diff
@@ -1,4 +1,10 @@
-## TACA - Tool for the Automation of Cleanup and Analyses
+<p align="center">
+  <a href="https://github.com/guillermo-carrasco/TACA">
```
Comment (guillermo-carrasco) [inline README.md]: btw that should be `<a href="https://github.com/SciLifeLab/TACA">`

File: requirements-dev.txt (hunk 1 above)
Comment (robinandeer) [inline requirements-dev.txt]: Thanks! I should put this in "software-development" then...

Just wanted to mention that you need to put this at the top of ``requirements.txt`` => ``-r requirements/conda.txt`` for it to make sense :sweat_smile:

**Comment by guillermo-carrasco** [comment]
What do you think @senthil10 , @ewels , @mariogiov ?

I'll merge this straight away if everyone likes the logo :-D

File: requirements-dev.txt (hunk 1 above)
Comment (senthil10) [inline requirements-dev.txt]: hey, but I have a question (maybe a stupid one). So these dependencies are needed to be installed (while developing) using `pip  install -r requirements-dev.txt` ?

**Comment by senthil10** [comment]
@guillermo-carrasco : I am happy with the logo :+1:  
nicely one @robinandeer :clap:

File: requirements-dev.txt (hunk 1 above)
Comment (robinandeer) [inline requirements-dev.txt]: No question is stupid. I had to install the sphinx dependencies to be able to build the docs locally. Also I remember that you use nose for running test, no?

Comment (guillermo-carrasco) [inline requirements-dev.txt]: I've told you: there are no stupid questions! ;-)

Yes these dependencies are not needed for TACA to work, but if you're developing and you want to run the tests locally, you need `nose`. If you want to `make` the docs before pushing changes to GitHub, then you will need `sphing`. Finally if you want you local docs to look exactly like [this](http://taca.readthedocs.org/en/latest/) you will need `sphinx-rtd-theme`.

Comment (senthil10) [inline requirements-dev.txt]: Ah ha,. Got it now. Tack @robinandeer and @guillermo-carrasco for the explanation :smile:

**Comment by robinandeer** [comment]
:bow:

File: requirements-dev.txt (hunk 1 above)
Comment (guillermo-carrasco) [inline requirements-dev.txt]: oh we wrote at the same time :sweat_smile:

Comment (robinandeer) [inline requirements-dev.txt]: Almost creepy... hehe :busts_in_silhouette:

**Pull request merged**